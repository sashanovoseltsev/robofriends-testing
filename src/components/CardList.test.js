import { shallow } from 'enzyme';
import React from 'react';
import CardList from './CardList';

const robots = [
  {
    id: 1,
    name: 'Sasha',
    email: 'sasha@gmail.com'
  },
  {
    id: 2,
    name: 'Misha',
    email: 'misha@gmail.com'
  },
  {
    id: 3,
    name: 'Kolya',
    email: 'kolya@gmail.com'
  }
]

it('expects to render CardList', () => {
  expect(shallow(<CardList robots={robots}/>)).toMatchSnapshot();
});