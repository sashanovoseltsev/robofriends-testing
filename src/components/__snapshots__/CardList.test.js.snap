// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to render CardList 1`] = `
<div>
  <Card
    email="sasha@gmail.com"
    id={1}
    key="0"
    name="Sasha"
  />
  <Card
    email="misha@gmail.com"
    id={2}
    key="1"
    name="Misha"
  />
  <Card
    email="kolya@gmail.com"
    id={3}
    key="2"
    name="Kolya"
  />
</div>
`;
